HW04.Rmd

```diff
@@ -1,126 +1,239 @@
-
-Assume that:
-
-* Winter = January-March
-* Spring = April-June
-* Summer = July-September
-* Fall = October-December
-
-```{r season}
-
-```
-
-
-# These are more open ended questions from here on out, you could look at making multiple summaries or types of plots. 
-
-### Are whites who are killed by guns more likely to die because of suicide or homicide? How does this compare to blacks and hispanics?
-
-```{r race}
-
-```
-
-(hint maybe looking at percentages would be good)
-```{r race_2}
-
-```
-
-
-### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
-
-```{r police}
-
-```
-
-Think about performing stats such as a t.test to see if differences you are plotting are statistically significant
-
-```{r stats?}
-
-
-```
-
-
-```{r police_2}
-
-```
-
-```{r stats_2?}
-
-
-```
-
-
-Finally, all together now:
-```{r police_3}
-
-```
-
-
-## Session info
-
-```{r}
-# always good to have this for reproducibility purposes
-devtools::session_info()
-```
-
+---
+title: "Exploring Gun Deaths in America"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning=FALSE, message=FALSE)
+```
+
+## Objectives
+* To explore gun death data curated by FiveThirtyEight [original data available here](https://github.com/fivethirtyeight/guns-data) by
+    * Month
+    * Intent
+    * Age and sex
+    * Education
+    * Season
+    * Intent by race
+    * Police-relatedness 
+
+## Read in the data
+
+The gun_deaths.csv is provided in the repository. 
+```{r load-data, message=FALSE, warning=FALSE}
+library(tidyverse)    # load tidyverse packages, including ggplot2
+library(viridis)      # color scheme
+library(scales)       # required for overlaying percentages over bar plots
+
+# set theme
+theme_set(theme_bw())
+
+my_colors <- c("#2a9d8f", "#E76F51", "#E9C46A", "#3a6b7e", "#F4A261")
+
+# read in the data
+gun_deaths <- read.csv('gun_deaths.csv', 
+                       na.strings=c("NotAvailable"), 
+                       colClasses = c("integer", "integer",  
+                                      "factor", "factor",  
+                                      "factor",  "integer",  
+                                      "factor", "integer",  
+                                      "factor",  "factor"))
+
+# I thought the following would work, and I can't figure out why it doesn't.
+# gun_deaths <- gun_deaths %>% mutate(race = replace(race, race == "Not Availabletive American/Not Availabletive Alaskan", "Native AmericaNative"))
+# It just replaces my string with NA, not the new string. :(
+
+# So I did this instead:
+gun_deaths <- gun_deaths %>% mutate(race = gsub("Not Availabletive American/Not Availabletive Alaskan", "Native American/Native Alaskan", race))
+gun_deaths <- gun_deaths %>% mutate(race = gsub("/", "/\n", race))
```

Comment:
That is awesome you changed the race name in the column. I ran the code, and I noticed that "/n" still popped up in the table. I changed the "/\n" to a "/" and then it appeared with only one slash (unless that was your intention).